src/ToDoDetail/ToDoDetail.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import classNames from 'classnames';
+import './ToDoDetail.css';
+
+type ToDoDetailProps = {
+  isNewTask?: boolean,
+  selectedTask: {
+    title: string,
+    description: string,
+    deadline?: Date
+  } | null
+} & React.ComponentProps<'div'>;
+
+const ToDoDetail = (props: ToDoDetailProps) => {
+  const { isNewTask, selectedTask, className, ...otherProps } = props;
+
+  if(isNewTask) {
+    return (
+      <div {...otherProps} className={classNames(className, "ToDoDetail")}>
+        <h2>Add New Task</h2>
+      </div>
+    );
+  }
+
+  if(!selectedTask) {
+    return (
+      <div {...otherProps} className={classNames(className, "ToDoDetail")}>
```

Comment:
```suggestion
      <div {...otherProps} className={classNames(className, "ToDoDetail")}>
```

consider making this a component that receives children.

Reply:
Resolved in acfbc24